example/.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"flutter_secure_storage","path":"D:\\\\Repos\\\\GitHub\\\\flutter_secure_storage\\\\flutter_secure_storage\\\\","dependencies":[]}],"android":[{"name":"flutter_secure_storage","path":"D:\\\\Repos\\\\GitHub\\\\flutter_secure_storage\\\\flutter_secure_storage\\\\","dependencies":[]}],"macos":[{"name":"flutter_secure_storage_macos","path":"D:\\\\Repos\\\\GitHub\\\\flutter_secure_storage\\\\flutter_secure_storage_macos\\\\","dependencies":[]}],"linux":[{"name":"flutter_secure_storage_linux","path":"D:\\\\Repos\\\\GitHub\\\\flutter_secure_storage\\\\flutter_secure_storage_linux\\\\","dependencies":[]}],"windows":[{"name":"flutter_secure_storage_windows","path":"D:\\\\Repos\\\\GitHub\\\\flutter_secure_storage\\\\flutter_secure_storage_windows\\\\","dependencies":[]}],"web":[{"name":"flutter_secure_storage_web","path":"D:\\\\Repos\\\\GitHub\\\\flutter_secure_storage\\\\flutter_secure_storage_web\\\\","dependencies":[]}]},"dependencyGraph":[{"name":"flutter_secure_storage","dependencies":[]},{"name":"flutter_secure_storage_linux","dependencies":[]},{"name":"flutter_secure_storage_macos","dependencies":[]},{"name":"flutter_secure_storage_web","dependencies":[]},{"name":"flutter_secure_storage_windows","dependencies":[]}],"date_created":"2021-07-23 10:05:24.527446","version":"2.2.3"}
```

Comment:
It seems like a lot of files have been checked in to VCS. Can you please check if all directories contain a correct .[gitignore](https://github.com/flutter/flutter/blob/master/.gitignore)?

Reply:
Actually it was that there were old .gitignore files that weren't including all of the new things in flutter that needed to be ignored. I removed all of the old ones from the sub directories so that the root .gitignore could be used in a single location to make it easier in the future to update.  There are still ones in things like the ios and android folders per flutter's recommendations on this.